CHANGELOG.md

```diff
@@ -19,6 +19,7 @@ The types of changes are:
 
 ### Fixed
 
+* Show a helpful error message if Docker daemon is not running during "fides deploy" [#1694](https://github.com/ethyca/fides/pull/1694)
```

Comment:
Your changes get bumped to the top of the list, eh? 😏

Reply:
🙃 moving on down